JobConfig/pions/PionsStage1_2020.fcl

```diff
@@ -0,0 +1,146 @@
+// Andrei Gaponenko, 2013
+// Jake Colston & Joseph Brock, 2015 (edit for cd3)
+
+#include "fcl/minimalMessageService.fcl"
+#include "fcl/standardProducers.fcl"
+#include "fcl/standardServices.fcl"
+
+BEGIN_PROLOG
+
+// Energy cuts on electron, positron, photon, and neutron.
+CutEPPhN : {
+    type : intersection
+    pars : [ { type: kineticEnergy cut: 100.0 },
+             { type: pdgId pars: [ 11, -11, 22, 2112 ] } ]
+}
+
+END_PROLOG
+
+# Give this job a name.
+process_name : pionsg4s1
+
+source : {
+    module_type : EmptyEvent
+    //maxEvents   : 1000000
+}
+
+services : {
+    message                : @local::default_message
+    TFileService           : { fileName : "nts.owner.pions-g4s1.version.sequencer.root" }
+    RandomNumberGenerator  : {defaultEngineKind: "MixMaxRng"}
+    GeometryService        : { inputFile      : "JobConfig/common/geom_baseline.txt"    }
+   
+    ConditionsService      : { conditionsfile : "Mu2eG4/test/conditions_01.txt"      }
+    GlobalConstantsService : { inputFile      : "Mu2eG4/test/globalConstants_01.txt" }
+    G4Helper               : {}
+    SeedService            : @local::automaticSeeds
+}
+
+physics : {
+
+    producers : {
+
+        generate: @local::PrimaryProtonGun
+
+        genCounter : { module_type : GenEventCounter }
+
+        g4run : {
+
+            module_type       : Mu2eG4
+            physics           : @local::mu2eg4DefaultPhysics
+            ResourceLimits    : @local::mu2eg4DefaultResourceLimits
+            TrajectoryControl : @local::mu2eg4NoTrajectories
+            debug             : @local::mu2eg4DefaultDebug
+            visualization     : @local::mu2eg4NoVisualization
+
+            generatorModuleLabel : generate
+
+            SDConfig : {
+                enableSD : [ virtualdetector ] // activate just the explicitly listed SDs
+                TimeVD   : { times : [] }
+            }
+
+            // Mu2eG4 cuts are evaluated in the listed order.
+            // The first match wins.  For the "intersection"
+            // type, the first false stops the evaluation.
+            // For the "union" type the first true stops the evaluation.
+            Mu2eG4StackingOnlyCut : @local::mu2eg4CutNeutrinos
+            Mu2eG4SteppingOnlyCut : @local::mu2eg4NoCut
+            Mu2eG4CommonCut : {
+                type : union
+                pars : [
+                    @local::CutEPPhN,
+                    { type: inVolume pars: [ HallAir ] },
+                    {
+                        type  : inVolume
+                        pars  : [ TS3Vacuum, TS4Vacuum, TS5Vacuum, DS1Vacuum, DS2Vacuum, DS3Vacuum ]
+                        write : vacuuas1
+                    }
+                ]
+            }
+        } // g4run
+
```

Comment:
As with the services, please use the standard definitions for modules, or even better for groups of modules.  Again the purposes is to minimize the points of maintenance in the future.